### ISSUE TITLE
File not closing after use inserting_pdf

### ISSUE THREAD

**Issue by MarikSx** [pymupdf/PyMuPDF#1552]
PyMuPDF 1.19.4

File not closing after use inserting_pdf, everything works fine on the 1.19.3 version.

```
import sys
import fitz
import os


def main():
    print(sys.version, "\n", sys.platform, "\n", fitz.__doc__)

    fn = 'test.pdf'
    src = fitz.open(fn)

    doc = fitz.open()
    doc.insert_pdf(src)
    doc.save("after_insert.pdf")
    doc.close()
    src.close()

    print(doc.is_closed)
    print(src.is_closed)

    os.remove(fn)


if __name__ == '__main__':
    main()
```

Output:

> 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] 
>  win32 
>  
> PyMuPDF 1.19.4: Python bindings for the MuPDF 1.19.0 library.
> Version date: 2022-01-01 00:00:01.
> Built for Python 3.10 on win32 (64-bit).
> 
> True
> True
> Traceback (most recent call last):
>   File "D:\test_fitz\main.py", line 25, in <module>
>     main()
>   File "D:\test_fitz\main.py", line 21, in main
>     os.remove(fn)
> PermissionError: [WinError 32] The process cannot access the file because it is being used by another process: 'test.pdf'
> 
> Process finished with exit code 1

**Comment by JorjMcKie** [comment]
Duplicate of #1417.
If urgent or you would like to test, I can put a pre-wheel v1.19.5 for Windows in this channel.

**Comment by mohamedshurie** [comment]
Was getting the same error, downgraded to 1.19.1

**Comment by JorjMcKie** [comment]
Fixed in v1.19.5 currently being uploaded.
